.env.example

```diff
@@ -1,4 +1,4 @@
 PORT=3001
 DEV_SERVER_PORT=3000
-TOKEN_SECRET=finalProject
+TOKEN_SECRET=NX"m:hC@YVp9$-w4
```

Comment:
If this is your real token secret on Heroku you should change it on Heroku as that information should remain private for security reasons. `.env.example` is just a template file to let other developers knows what environment variables the application needs, so any dummy value is fine.